src/main/resources/jenkins/plugins/http_request/HttpRequestStep/help-acceptType.html

```diff
@@ -0,0 +1,3 @@
+<div>
+	Add 'Accept: foo' HTTP request header where 'foo' is the HTTP content type to accept.
+</div>
```

Comment:
Good practice. Add line breaks to the end of all files

Reply:
@oleg-nenashev Thanks for your feedback, line breaks are added now on all the files.